video/video.ino

```diff
@@ -202,6 +207,10 @@ void print(char const * text) {
 	for (auto i = 0; i < strlen(text); i++) {
 		processor->vdu(text[i]);
 	}
+	#if ZDI==1
+		// echo ESP32 characters to the debug host
+		DBGSerial.print(text);
+	#endif

```

Comment:
if you patch the calls inside `vdu.h` as I suggested above then this echo can be deleted

Reply:
done